lib/core/datatable/components/datatable/datatable.component.scss

```diff
@@ -441,7 +441,6 @@
             &.adf-datatable-cell-header,
             .adf-datatable-content-cell {
                 max-width: calc(100% - 0.1px);
-                overflow: hidden;
```

Comment:
how removing overflow fixes the tooltip?

Reply:
And also the ellipsis 'effect' would no longer work without the `overflow: hidden`. I've checked and long strings in the header cell would no longer have ellipsis.